internal/fetch/jobDescription_test.go

```diff
@@ -33,14 +39,54 @@ func Test_parseJobDescription(t *testing.T) {
 			},
 		},
 	} {
-		t.Run(tc.jobDescription, func(t *testing.T) {
+		s.Run(tc.jobDescription, func() {
 			got, err := ParseJobDescription(tc.defaultPipelineID, tc.jobDescription)
 			if tc.wantErr {
-				require.Error(t, err, tc.jobDescription)
+				s.Require().Error(err, tc.jobDescription)
 				return
 			}
-			require.NoError(t, err, tc.jobDescription)
-			assert.Equal(t, tc.want, got, tc.jobDescription)
+			s.Require().NoError(err, tc.jobDescription)
+			s.Assert().Equal(tc.want, got, tc.jobDescription)
+		})
+	}
+}
+
+func (s *JobDescriptionSuite) TestJobDescription_External() {
+	for _, tc := range []struct {
+		name           string
+		jobDescription JobDescription
+	}{
+		{
+			name: "sd@123:fooBar",
+			jobDescription: JobDescription{
+				PipelineID: 123,
+				JobName:    "fooBar",
+			},
+		},
+	} {
+		s.Run(tc.name, func() {
```

Comment:
I would prefer to separate the `tc` declaration / initialization and the `for` loop for better readability and clarity:
```
tests := []struct {
  // struct fields
} {
   {name: "foo", jobDescription: /* etc */ },
   // fill in the slice with test instances
} // done with tests "creation"! Deep breath... :)

for _, tt := range tests {
  // run tests
}

Reply:
Done